packages/database/src/migrations/20210507014219-PutWaiviviaToTheRightPlace-modifies-data.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const hierarchyNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM entity_hierarchy WHERE name = '${name}';`);
+  return record?.rows[0].id;
+};
+
+const WAIVIVIA_CODE = 'FJ_WAV';
+const BURETA_CODE = 'FJ_CATCH_Bureta';
+const FIJI_CODE = 'FJ';
+const EASTERN_CODE = 'FJ_Eastern';
+const projectCode = 'wish';
+
+exports.up = async function (db) {
+  const WAIVIVIA_ID = await codeToId(db, 'entity', WAIVIVIA_CODE);
+  const BURETA_ID = await codeToId(db, 'entity', BURETA_CODE);
+  const FIJI_ID = await codeToId(db, 'entity', FIJI_CODE);
+
```

Comment:
I was more thinking that the village and catchment were key to the input below if not found it would insert a row to the entity_relation table with child_id: null for example do something like this:

```suggestion
if 
throw new Error('MapOverlayGroup not found');
```

Reply:
Good idea. My only concern is that, does throwing error will stop running the following migrations? If so, instead of throwing error, I would skip the sql.insert if they are null id.

Reply:
I will leave up to you. My thinking is that if the parent isn't found then the migration has failed and would be good to know that rather than a silent fail. But as this is a small change and easily checked i don't want to be too nick picky :)

Reply:
Thanks @nick-beyondessential for raising this question in dev channel, I didn't think of it before. Throwing error in migrations makes more sense to me now.